tests/integration_test/services_test.py

```
@@ -6904,6 +6904,312 @@
+
+    for secret in secrets:
+        LLMSecret(**secret).save()
+
+    payload = {
+      "connector_type": "petpooja",
+      "config": {
+        "restaurant_name": "restaurant1",
+        "branch_name": "branch1",
+        "restaurant_id": "98765"
+       },
+       "meta_config": {
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)